src/Model/Message.php

```diff
@@ -99,11 +98,11 @@ public static function createRequestForQuery(Query $query)
     /**
      * Creates a new response message for the given query with the given answer records
      *
-     * @param Query    $query
-     * @param Record[] $answers
+     * @param Query $query
+     * @param array<Record> $answers
```

Comment:
Wouldn't have mentioned this if it wasn't for https://github.com/reactphp/dns/pull/225/files#diff-5425bc2f31058fccb9ce30234ca1c36e96a98a6347cfd5caa235a6ea66bb98a4L139-R143, I think we should align this here as well as it was also aligned before

Reply:
It's unclear to me what the alignment is as those lines in Record are about a union type